package.json

```diff
@@ -6,10 +6,18 @@
   "author": "Maksim Sink <maksim@sinik.it>",
   "license": "MIT",
   "private": true,
+  "scripts": {
+    "test": "exit 0",
+    "clean": "rimraf packages/*/node_modules packages/*/dist packages/*/build node_modules",
+    "download": "wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.8/swagger-codegen-cli-2.4.8.jar -d ./bin/",
+    "build:core": "npm run download && npm run build:swagger && java -Dmodels -jar ./bin/swagger-codegen-cli-2.4.8.jar generate -i ./build/swagger.yml -l typescript-inversify -o ./packages/core",
```

Comment:
looks like these scripts require java to be executed... 

I don't think this is an issue, but it might be helpful if there is some prerequisite documentation that a Java download gets added to. 

Thoughts?

Reply:
It may not be the final generator it's just there as an example. Check out the list of projects under "interesting projects".